tutormfe/patches/openedx-lms-production-settings

```diff
@@ -28,7 +28,7 @@ MFE_CONFIG = {
 {% if get_mfe("authn") %}
 AUTHN_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ MFE_HOST }}/authn"
 AUTHN_MICROFRONTEND_DOMAIN  = "{{ MFE_HOST }}/authn"
-MFE_CONFIG["DISABLE_ENTERPRISE_LOGIN"] = True
+MFE_CONFIG["DISABLE_ENTERPRISE_LOGIN"] = "true"
```

Comment:
Same here. Can you please share why are we converting to string here as well? I checked this [here](https://github.com/search?q=repo%3Aopenedx%2Ffrontend-app-authn%20DISABLE_ENTERPRISE_LOGIN&type=code). There is no comparison done against a string.